client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -152,32 +152,35 @@ export class SeoForm extends React.Component {
 		}
 
 		// Don't update state for fields the user has edited
-		nextState = omit( nextState, dirtyFields.toArray() );
+		nextState = omit( nextState, [ ...dirtyFields ] );
```

Comment:
very interesting casting! 
I wasn't sure exactly what `babeljs` would convert this into. I [checked on babeljs](https://babeljs.io/repl/#?babili=false&browsers=&build=&builtIns=false&code_lz=MYewdgzgLgBAhjAvDMBTA7jAyqqAKASgG4AoOAOjgBMq8YByACVQBsWQYB1EAJxavoxiJEqEiwARkhgBtcvLgBdUqPAQQLVOXYBzOlIKrIGrbroBBHjzgBPcgDMeIALZ44BIUA&debug=false&circleciRepo=&evaluate=true&lineWrap=false&presets=es2015%2Ces2016%2Ces2017%2Creact%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3%2Ces2015-loose&prettier=false&targets=&version=6.26.0) and it looks like it transforms into:
`[].concat( dirtyFields )` which won't actually be correct.

What we want here is:
`omit( nextState, Array.from( dirtyFields ) )`

Reply:
In actual testing...this worked.  For some reason our configuration of babel is different than that web repl and it will call `toConsumableArray` on the Set making the end result correct 🤔 .

Reply:
Problematic is that `Array.from()` isn't in IE. The `Set` returns an `Iterator` which is why I used the spread operator, as it casts an iterable into the array.

Reply:
> Array.from()

Hopefully we polyfill it! We have a number of uses throughout the codebase!

Reply:
adapted in 58553611a4